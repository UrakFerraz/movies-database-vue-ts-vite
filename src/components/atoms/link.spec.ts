import { mount } from '@vue/test-utils'
import Link from './link.vue';

const props = {
  class: 'test-link',
  href: 'https://www.google.com',
  target: '_blank',
  title: 'test link'
	}

const wrapper = mount(Link, {props});

describe('deve ser um componente link', () => {
	it('deve renderizar tag <a>', () => {
		expect(wrapper.html()).toBeDefined
		expect(wrapper.html()).toMatchSnapshot()
		expect(wrapper.html()).toBe(`<a class="test-link" href="https://www.google.com" target="_blank">test link</a>`)
	})
})
