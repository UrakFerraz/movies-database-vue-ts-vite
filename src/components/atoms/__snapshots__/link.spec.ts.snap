// Vitest Snapshot v1

exports[`deve ser um componente link > deve renderizar tag <a> 1`] = `"<a class=\\"test-link\\" href=\\"https://www.google.com\\" target=\\"_blank\\">test link</a>"`;
